sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/ColumnDefinition.scala

```diff
@@ -51,14 +54,22 @@ case class ColumnDefinition(
   }
 
   def toV2Column(statement: String): V2Column = {
```

Comment:
What's the difference between V1 column and V2 column? Which one should we support?